x-pack/filebeat/module/aws/cloudwatch/ingest/pipeline.yml

```diff
@@ -0,0 +1,26 @@
+description: "Pipeline for s3 server access logs"
+
+processors:
+  - grok:
+      field: message
+      patterns:
+        - >-
+          %{TIMESTAMP_ISO8601:_tmp.timestamp} %{MONTH:_tmp.month} %{MONTHDAY:_tmp.day} %{TIME:_tmp.time} %{IPORHOST:aws.cloudwatch.ip} %{NOTSPACE:aws.cloudwatch.program_name}: %{GREEDYDATA:message}
```

Comment:
The field `aws.cloudwatch.ip` suggest it's only IP address, but according to the field definition it's also host. Is there any other field name that would cover both cases? Maybe `aws.cloudwatch.source` ?

Reply:
Thanks! `aws.cloudwatch.source` sounds better!

Reply:
For example in `2020-02-20T07:01:01.000Z Feb 20 07:01:01 ip-172-31-81-156 systemd: Stopping User Slice of root.`, the ip part is `ip-172-31-81-156`. It's not in `172.31.81.156` format and that's why I chose IPORHOST to parse the field. So it covers a-z and -.

Reply:
That's alright ;) I was just referring to the field name, field def is ok.

Reply:
Thanks for the review!!